docs/sources/reference/playlist.md

```diff
@@ -11,59 +11,96 @@ weight = 4
 
 ## Create a playlist
 
 {{< docs-imagebox img="/img/docs/v50/playlist.png" max-width="25rem" class="docs-image--right">}}
 
-The Playlist feature can be accessed from Grafana's sidemenu, in the Dashboard submenu.
+To create a playlist: 
+
+1. Click the __New playlist__ button.
+1. Name your playlist in the __Name__ text box.
+1. Enter a time interval in the __Interval__ text box.
+
+The time interval is the amount of time for Grafana to stay on a particular dashboard before advancing to the next one on the playlist.
+
+## Add a dashboard to a playlist
 
-Click the **New playlist** button to create a new playlist. Firstly, name your playlist and configure a time interval for Grafana to wait on a particular Dashboard before advancing to the next one on the Playlist.
+To add a dashboard:
 
-You can search Dashboards by name (or use a regular expression), and add them to your Playlist. Or you could add tags which will include all the dashboards that belongs to a tag when the playlist start playing. By default, your starred dashboards will appear as candidates for the Playlist.
+1. Click the __Find dashboards by name__ text box.
+1. Search for the playlist by name or regular expression.
+   - You can also filter your results by starred status or tags.
 
-Be sure to click the **Add to playlist** next to the dashboard name to add it to the Playlist. To remove a dashboard from the playlist click on **Remove[x]** from the playlist.
+     By default, your starred dashboards will appear as options to add to the Playlist.
+1. Click __Add to playlist__ next to the dashboard you want to add.
+   - Click on __Remove[x]__ to remove a dashboard from the playlist.
 
-Since the Playlist is basically a list of Dashboards, ensure that all the Dashboards you want to appear in your Playlist are added here.
+The Playlist is essentially a list of dashboards. Be sure that all the dashboards you want to appear in your playlist are added in this section before saving the playlist.
 
 ## Save a playlist
 
-Once all the wanted dashboards are added to a playlist, you can save this playlist by clicking on the green **Save** button. This will generate a unique URL for you playlist which can be shared if needed. Click on the generated URL or on the **Play** button from the **Saved playlists** list to start the playlist. If you want to share the URL, right click on the URL and copy the URL link and share.
+To save a playlist: 
+
+1. Ensure that your playlist has a __Name__, __Interval__, and at least one __Dashboard__ added to it.
+1. Click the green __Create__ button.
+
+   This will generate a unique, shareable URL for your playlist.
+
+ Click on the generated URL to start the Playlist from here. If you want to share the URL, right click on the URL and copy the URL link and share.
 
 ## Start a playlist
 
-Also, if you want, you can start the playlist without saving it by clicking on the green **Start** button at the bottom.
+To start a playlist: 
```

Comment:
See previous comments about first paragraph. 

This would be a good place to state that to start a playlist, you must have at least one playlist already created per (link to Create a playlist).